hv.c

```diff
@@ -1257,11 +1257,13 @@ S_hv_delete_common(pTHX_ HV *hv, SV *keysv, const char *key, STRLEN klen,
                             " a restricted hash");
         }
 
+        sv = HeVAL(entry);
```

Comment:
> Moving this up another 10 lines would allow another 3 `HeVAL()` entries to be replaced.

Why didn't I notice that? :-(

Done - definitely an improvement.

> Note also there's a typo in the commit message: `s/HvVAL/HeVAL/`.

Thanks. Fixed.

I've fixed both of these, rebased on blead and pushed the branch again.